src/app/components/flyout/flyout-guidelines.component.html

```diff
@@ -127,7 +127,7 @@
 	
 	<stache-row>
     <stache-column screenSmall="6">
-			<p>Do not use a flyout for tasks where users always need a full-screen view of details, such as expanded views of tiles. Navigate to a full page instead.</p>
+			<p>Do not use a flyout for tasks where users will need a full-screen view of details or will be taking multiple actions within the view. Navigate to a full page instead.</p>
```

Comment:
Because we're trying to move away from using "do not," maybe something like:

Use a full a page when users need a full-screen view of details or when users can perform multiple actions within the view.

Reply:
I'd make similar changes here to use present tense verbs: "Do not use a flyout for tasks where users require a full-screen view of details or need to take multiple actions in the view..."